Sources/Puree/TagPattern.swift

```diff
@@ -4,13 +4,17 @@ private let separator: Character = "."
 private let allWildcard = "**"
 private let wildcard = "*"
 
-public struct TagPattern {
+public struct TagPattern: CustomStringConvertible {
```

Comment:
Doesn't look like you're using it anywhere in this PR...

Reply:
Sure.

I added on this opportunity.